onnxruntime/core/providers/openvino/ov_interface.h

```diff
@@ -0,0 +1,114 @@
+// Copyright(C) 2019 Intel Corporation
+// Licensed under the MIT License
+
+#pragma once
+
+#include <inference_engine.hpp>
+#include <openvino/core/core.hpp>
+#include <openvino/runtime/runtime.hpp>
+
+namespace onnxruntime {
+
+namespace openvino_ep {
+
+class ov_core;
```

Comment:
I think we should have a defined naming scheme.
-> As we are naming all variables (first word all lower case, subsequent words starting with upper case). which is the conventional way.

-> Similarly, lets have all the class names first letter to be capitalized
(Class names: concatenated words each starting with upper case)